src/main/java/ch/ethz/seb/sebserver/webservice/weblayer/api/LightController.java

```diff
@@ -0,0 +1,137 @@
+import ch.ethz.seb.sebserver.gbl.async.AsyncServiceSpringConfig;
+import ch.ethz.seb.sebserver.gbl.model.Domain;
+import ch.ethz.seb.sebserver.gbl.profile.WebServiceProfile;
+import ch.ethz.seb.sebserver.gbl.util.Utils;
+import ch.ethz.seb.sebserver.webservice.servicelayer.dao.AdditionalAttributesDAO;
+import ch.ethz.seb.sebserver.webservice.servicelayer.dao.FilterMap;
+import ch.ethz.seb.sebserver.webservice.servicelayer.dao.SEBClientConfigDAO;
+import ch.ethz.seb.sebserver.webservice.servicelayer.session.SEBClientConnectionService;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executor;
+
+@WebServiceProfile
+@RestController
+@RequestMapping("${sebserver.webservice.api.exam.endpoint.discovery}")
+@ConditionalOnExpression("'${sebserver.webservice.light.setup}'.equals('true')")
+public class LightController {
+
+    private final SEBClientConnectionService sebClientConnectionService;
+    private final SEBClientConfigDAO sebClientConfigDAO;
+    private final AdditionalAttributesDAO additionalAttributesDAO;
+    private final Executor executor;
+
+    protected LightController(
+            final SEBClientConnectionService sebClientConnectionService,
+            final SEBClientConfigDAO sebClientConfigDAO,
+            final AdditionalAttributesDAO additionalAttributesDAO,
+            @Qualifier(AsyncServiceSpringConfig.EXAM_API_EXECUTOR_BEAN_NAME) final Executor executor) {
+
+        this.sebClientConnectionService = sebClientConnectionService;
+        this.sebClientConfigDAO = sebClientConfigDAO;
+        this.additionalAttributesDAO = additionalAttributesDAO;
+        this.executor = executor;
+    }
+
+    //this.examAPI_V1_Endpoint + API.EXAM_API_CONFIGURATION_LIGHT_ENDPOINT
+    //http://localhost:8080/exam-api/discovery/light-config
+    @RequestMapping(
+            path =  API.EXAM_API_CONFIGURATION_LIGHT_ENDPOINT,
+            method = RequestMethod.GET,
+            produces = MediaType.APPLICATION_OCTET_STREAM_VALUE)
+    public CompletableFuture<Void> getLightConfig(
+            final HttpServletRequest request,
+            final HttpServletResponse response){
+
+        //temp solution: get first active seb client config we can get -->
+        //in a light setup there should be only one setup so this step is not necessary and we can just use the first and only item in the db
+        String modelId = getSebClientConfigId();
```

Comment:
Dieser Schritt ist wahrscheinlich nicht notwendig, da die Funktion "getSebClientConfigId" eine Liste holt und den die ID des ersten Eintrag nimmt --> Wir können einfach 1 nehmen, da in einem SSL-Setup sowieso nur eine Connection Config existiert